src/menuFactory.js

```diff
@@ -194,33 +203,42 @@ export default (styles) => {
     },
 
     render() {
-      let items, svg, overlay;
```

Comment:
I really like the idea of making this render function cleaner, but if these are going to be ternaries, we might as well inline it all in the JSX :)

I have done this in v1.8.4, so thanks for the idea!